### PR TITLE
Add project as an attribute on job entity.

### PR DESCRIPTION
**Pull request by inferno-chromium**
This is needed for dynamic filters on UI e.g. fuzzer stats.

### CODE REVIEW

File: src/appengine/handlers/jobs.py [hunk 1]
```diff
@@ -127,6 +131,7 @@ def post(self):
     job.description = description
     job.environment_string = environment_string
     job.templates = templates
+    job.project = project
```
Comment (oliverchang) [inline src/appengine/handlers/jobs.py]: How about setting this as a `_pre_put_hook` in data_types.Job? That will be a bit more reliable, and can prevent issues in the future when we write e.g. migration scripts.

Comment (inferno-chromium) [reply src/appengine/handlers/jobs.py]: Great idea, forgot about hook. done!

Comment (oliverchang) [reply src/appengine/handlers/jobs.py]: this line needs to be removed now?

Comment (inferno-chromium) [reply src/appengine/handlers/jobs.py]: yes fixed after last patch

**Pull request merged**